script.js

```diff
@@ -0,0 +1,49 @@
+const bklist = document.querySelector('.bklist');
+const bkTitle = document.getElementById('title');
+const bkAuthor = document.getElementById('author');
+const form = document.getElementById('form');
+let id = 0;
+let books = [];
+if (localStorage.getItem('Data-base') !== null) {
+  books = JSON.parse(localStorage.getItem('Data-base'));
+}
```

Comment:
- [ ] Let's make sure that on-page refresh the items in the localStorage are still displayed on the page. You may use the [DOMContentLoaded](https://developer.mozilla.org/fr/docs/Web/API/Window/DOMContentLoaded_event) event to display the books stored in the LocalStorage.

Reply:
Hello @EricMbouwe thank you for this review.
We had to refactor everything and it now works as you have suggested. The only small issue is that the remove button visually removes the book when you refresh the page. But behind the scenes in the object and in the local storage, it removes.
I really hope I won't get flagged because of this.